dir.props

```diff
@@ -140,6 +140,10 @@
     <PackageThirdPartyNoticesFile>$(ProjectDir)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
     <SyncInfoDirectory>$(BaseIntermediateOutputPath)</SyncInfoDirectory>
 
+    <!-- If true, indicates that this is not an officially supported release -->
+    <!-- It is important to flip this to false in official release branches -->
+    <IsPrerelease>true</IsPrerelease>
```

Comment:
So we need to flip both this bit and the one in clrdefinitions.cmake? Is there a good way too hook the two up together? If not can you please add a comment in both places pointing at the other so that they should be flipped together.

Reply:
I agree. Please at least add a comment.

Reply:
> Is there a good way too hook the two up together?

I couldn't find one, but I don't know much about CoreCLR build. Is there someone who's a CoreCLR build subject matter expert who could tell us?

Reply:
There is not good way to link these together. The managed and unmanaged builds have to be kept in sync manually.